### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by jeffandersen**

### CODE REVIEW

**Comment by jeffandersen** [comment]
I think this would really help your website.

**Comment by jeffandersen** [comment]
Aw shucks :(

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 		<meta charset="UTF-8">
 
 
-		<title>Cleveland Data Solutions</title>
+		<title>Jeff's Data Solutions</title>
```
Comment (jeffandersen) [inline index.html:7]: This shouldn't say this!

**Comment by chriskol** [comment]
I think you should do it!

**Pull request closed without merging**